release-team/role-handbooks/communications/release-blog.md

```diff
@@ -104,7 +104,7 @@ Check out the full details of the Kubernetes 1.XX release in our [release notes]
 
 ### Availability
 
-Kubernetes 1.XX is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.XX.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters using Docker container “nodes” with [kind](https://kind.sigs.k8s.io/). You can also easily install 1.XX using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
+Kubernetes 1.XX is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.XX.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters using Docker container “nodes” with [minikube](https://minikube.sigs.k8s.io/). You can also easily install 1.XX using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
```

Comment:
1- for user's quicker experience we could mention --kubernetes-version flag (so user tries this version of Kubernetes)

2- also we dont need to mention docker here, since minikube works with docker or any other driver.

Reply:
I removed the reference to the docker container

The release notes are pretty short and concise though and I'm not sure if adding a usage example would make the most sense in this scenario